### DISCUSSION TITLE
Web app that allows any ESPN Fantasy football league to see historical stats

### DISCUSSION THREAD

**Discussion by zanderman12** [cwendt94/espn-api#286] [Show and tell] [answered]
Wanted to share a web app I've been working on: fantasyleaguereport.com

It allows users to input their league info (including s2 and swid cookies if private league) then the app will pull a bunch of historical stats like season standings, head-to-head records, coach and drafting ability etc.  Also generates weekly emails with personalized matchup recaps, power rankings, and luck scores.

Thank you cwendt for this package, really could not have built fantasy league report without it

**Comment by cwendt94** [comment, marked as answer]
This is great, thanks for sharing!
